go.mod

```diff
@@ -31,7 +31,7 @@ require (
 
 require (
 	github.com/IBM/platform-services-go-sdk v0.59.1
-	github.com/golang-jwt/jwt/v4 v4.5.0
+	github.com/golang-jwt/jwt/v4 v4.5.1
```

Comment:
This should be v4.5.2 see: https://github.com/golang-jwt/jwt/security/advisories/GHSA-mh63-6h87-95cp

Reply:
And https://nvd.nist.gov/vuln/detail/CVE-2025-30204

Reply:
cc. @aman4433